src/brsTypes/BrsType.ts

```diff
@@ -90,6 +92,48 @@ export namespace ValueKind {
     }
 }
 
+/**
+ *  Converts a specified brightscript type in string into BrsType representation, with actual value
+ *  Note: only supports native types so far.  Objects such as array/AA aren't handled at the moment.
+ *  @param {string} type data type of field
+ *  @param {string} value optional value specified as string
+ *  @returns {BrsType} BrsType value representation of the type
+ */
+export function getBrsTypeFromString(type: string, value?: string): BrsType {
+    let returnValue: BrsType;
+
+    switch (type.toLowerCase()) {
+        case "boolean":
+            returnValue = value ? BrsBoolean.from(value === "true") : BrsBoolean.False;
+            break;
+        case "dynamic":
+            returnValue = BrsInvalid.Instance;
```

Comment:
Is it valid to define a type as `dynamic` in scenegraph xml? I thought that was only in brightscript code

Reply:
You're right, `dynamic` is not a valid type [according to the docs](https://developer.roku.com/docs/references/scenegraph/xml-elements/interface.md) but on the other hand [node](https://developer.roku.com/docs/references/scenegraph/abstract-nodes/node.md) doesn't define a type for `focusedChild` field. I think I'll update that case to accept `"invalid"` for those edge cases

Reply:
While the docs are weird, let's change this to "node" as that seems what _focusedChild_ is in this case.  Let's also update that in the constructor of roSGNode.